inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -994,13 +993,13 @@ survey_design %>%                  # use the survey object (weighted)
     # (this could also be "yes" if your variables were yes/no; 
     # or a combination, e.g. c("yes", TRUE, "sick"))
     # method xlogit uses the CDC SUDAAN method of calculating confidence intervals
-  tabulate_binary_survey(contains("symptoms"), keep = "", invert = TRUE, method = "xlogit") %>%
+  tabulate_binary_survey(contains("symptoms"), keep = "", invert = TRUE, method = "xlogit", digits = 2) %>%
```

Comment:
did you mean to have digits duplicated? is here and kable now... same in chunks below

Reply:
Yes, I did mean for this. The function automatically combines the proportion and CI columns, which uses the digits argument, but the `n` columns are left as numbers, so that's what the kable digits provides. 

We *could* also just specify `options(digits = 2)` at the beginning of the document.